app/models/hmpps_api/offender_base.rb

```diff
@@ -231,7 +239,7 @@ def ldu
     end
 
     def handover
-      @handover ||= if pom_responsibility.custody?
+      @handover ||= if pom_responsibility.responsible?
                       HandoverDateService.handover(self)
                     else
                       HandoverDateService::NO_HANDOVER_DATE
```

Comment:
It seems a shame to still be blanking out handover dates when the Community is responsible.

Not sure it's something to tackle in this PR, but just pointing it out. Maybe one to raise and clean up later if this behaviour is no longer desirable.